estate/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    'name': "Estate",
```

Comment:
Be consistent with ' or ", doesn't matter which but keep the same across the file.